in-market-test/v2/ansible/build_network_file.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import configparser
+import os
+import re
+import sys
+from pathlib import Path
+
+
+def get_urls_from_config():
```

Comment:
please specify the type hints at least for the functions

Reply:
added.